src/questFolder/gatherVatMaggots.py

```diff
@@ -126,10 +126,14 @@ def getNextStep(self, character=None, ignoreCommands=False, dryRun = True):
         self.fail(reason="no tree")
         return (None,None)
     @staticmethod
-    def generateDutyQuest(beUsefull,character,currentRoom):
+    def generateDutyQuest(beUsefull,character,currentRoom, dryRun):
         terrain = character.getTerrain()
+        try:
```

Comment:
stuff like this is to not break old savestate and can be remove most of the time